{
  "CVE Code": "CVE-2024-5629",
  "Published": "2024-06-05T14:32:56.435Z",
  "Updated": "2025-02-13T17:54:22.106Z",
  "Description": "An out-of-bounds read in the 'bson' module of PyMongo 4.6.2 or earlier allows deserialization of malformed BSON provided by a Server to raise an exception which may contain arbitrary application memory.",
  "ProblemTypes": [
    "CWE-125: Out-of-bounds Read"
  ],
  "Affected": [
    {
      "vendor": "MongoDB Inc",
      "product": "PyMongo",
      "versions": [
        "0"
      ]
    }
  ],
  "Reference": [
    "https://jira.mongodb.org/browse/PYTHON-4305",
    "https://lists.debian.org/debian-lts-announce/2024/06/msg00007.html"
  ]
}